src/utils/testing-utils.tsx

```diff
@@ -1,9 +1,25 @@
-/* eslint-disable import/no-extraneous-dependencies */
-import { screen } from '@testing-library/react';
+import React, { ReactElement } from 'react';
+// eslint-disable-next-line import/no-extraneous-dependencies
```

Comment:
To be honest, this file is supposed to be only for testing.
I don't really care leaving the whole file with extraneous dependencies

Reply:
Oops you're right I don't remember why I moved the eslint-disabled to be just for this line, will change it back